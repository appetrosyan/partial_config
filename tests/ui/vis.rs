#[derive(partial_config::HasPartial)]
pub(crate) struct ConfigurationWarn {

}

fn main() {}
