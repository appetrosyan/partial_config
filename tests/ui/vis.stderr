error: Cannot implement `HasPartial` for a private structure.

         = help: If your structure is private, it is better to convert to it with an `Into::into` rather than directly derive `HasPartial`, which by definition will expose some of the fields

 --> tests/ui/vis.rs:2:1
  |
2 | pub(crate) struct ConfigurationWarn {
  | ^^^^^^^^^^
